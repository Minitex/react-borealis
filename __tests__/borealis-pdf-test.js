import React from 'react'
import renderer from 'react-test-renderer'
import BorealisPDF from '../src/borealis-pdf'

describe('BorealisPDF Tests', () => {

  it("renders an audio", function(){  
    let items = [
      {
        "type": "pdf",
        "label": "PDF",
        "focus": true,
        "src": "https://s3.amazonaws.com/mdl-assets/stpaul.pdf",
      },
      {
        "type": "transcript",
        "label": "Transcript",
        "text": "PDF Transcript Here",
        "focus": false
      }
    ]
    const component = renderer.create(
      <BorealisPDF items={items} />
    )
    let tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })
})