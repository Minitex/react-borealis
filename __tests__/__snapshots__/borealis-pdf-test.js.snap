exports[`BorealisPDF Tests renders an audio 1`] = `
<div>
  <div
    className="row">
    <div
      className="row image-nav">
      <ul
        className="nav nav-pills">
        <li
          className="active"
          onClick={[Function]}
          role="presentation">
          <a
            href="">
            PDF
          </a>
        </li>
        <li
          className=""
          onClick={[Function]}
          role="presentation">
          <a
            href="">
            Transcript
          </a>
        </li>
      </ul>
    </div>
  </div>
  <div
    className="row">
    <object
      data="https://s3.amazonaws.com/mdl-assets/stpaul.pdf"
      height={500}
      type="application/pdf"
      width={800}>
      <img
        src={undefined} />
       
      <a
        href="https://s3.amazonaws.com/mdl-assets/stpaul.pdf">
        <span
          className="glyphicon glyphicon-download-alt" />
         
        https://s3.amazonaws.com/mdl-assets/stpaul.pdf
      </a>
    </object>
  </div>
</div>
`;
